From 781423cc09466fb2e4cbe0bd9ee3dce1318d9d88 Mon Sep 17 00:00:00 2001
From: Ethan Burns <burns.ethan@gmail.com>
Date: Sun, 25 Sep 2016 10:48:12 -0400
Subject: [PATCH] acme: change color scheme.

---
 src/cmd/acme/acme.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/cmd/acme/acme.c b/src/cmd/acme/acme.c
index a08de9b..636e515 100644
--- a/src/cmd/acme/acme.c
+++ b/src/cmd/acme/acme.c
@@ -971,16 +971,16 @@ iconinit(void)
 
 	if(tagcols[BACK] == nil) {
 		/* Blue */
-		tagcols[BACK] = allocimagemix(display, DPalebluegreen, DWhite);
-		tagcols[HIGH] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DPalegreygreen);
-		tagcols[BORD] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DPurpleblue);
+		tagcols[BACK] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, 0xf0f0f0ff);
+		tagcols[HIGH] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, 0xb6dafdff);
+		tagcols[BORD] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DBlack);
 		tagcols[TEXT] = display->black;
 		tagcols[HTEXT] = display->black;
 	
 		/* Yellow */
-		textcols[BACK] = allocimagemix(display, DPaleyellow, DWhite);
-		textcols[HIGH] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DDarkyellow);
-		textcols[BORD] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DYellowgreen);
+		textcols[BACK] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, 0xfaf0e6ff);
+		textcols[HIGH] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, 0xb6dafdff);
+		textcols[BORD] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, 0xaaaaaaff);
 		textcols[TEXT] = display->black;
 		textcols[HTEXT] = display->black;
 	}
-- 
2.9.3

