From 7a8f76b8993736bdfcd0928cc51499be7642bfa1 Mon Sep 17 00:00:00 2001
From: Ethan Burns <eaburns@google.com>
Date: Tue, 12 May 2015 10:13:11 -0400
Subject: [PATCH] On Del, grow top window down, not bottom window up.

---
 src/cmd/acme/cols.c | 24 +++++++++++++++---------
 1 file changed, 15 insertions(+), 9 deletions(-)

diff --git a/src/cmd/acme/cols.c b/src/cmd/acme/cols.c
index 5e16c21..da9042d 100644
--- a/src/cmd/acme/cols.c
+++ b/src/cmd/acme/cols.c
@@ -157,7 +157,8 @@ void
 colclose(Column *c, Window *w, int dofree)
 {
 	Rectangle r;
-	int i, didmouse, up;
+	int i, didmouse;
+	Window *wdel = nil;
 
 	/* w is locked */
 	if(!c->safe)
@@ -183,23 +184,28 @@ colclose(Column *c, Window *w, int dofree)
 		draw(screen, r, display->white, nil, ZP);
 		return;
 	}
-	up = 0;
-	if(i == c->nw){		/* extend last window down */
+	if(i > 0){		/* extend last window down */
 		w = c->w[i-1];
 		r.min.y = w->r.min.y;
-		r.max.y = c->r.max.y;
+		if (i == c->nw) {
+			r.max.y = c->r.max.y;
+		} else {
+			wdel = c->w[i];
+			// Leave room for the 2px border.
+			r.max.y = c->w[i]->r.min.y - 2;
+		}
 	}else{			/* extend next window up */
-		up = 1;
 		w = c->w[i];
+		wdel = w;
 		r.max.y = w->r.max.y;
 	}
 	draw(screen, r, textcols[BACK], nil, ZP);
 	if(c->safe) {
-		if(!didmouse && up)
-			w->showdel = TRUE;
+		if(!didmouse && wdel)
+			wdel->showdel = TRUE;
 		winresize(w, r, FALSE, TRUE);
-		if(!didmouse && up)
-			movetodel(w);
+		if(!didmouse && wdel)
+			movetodel(wdel);
 	}
 }
 
-- 
2.9.3

